purldb-toolkit/README.rst

```diff
@@ -1,12 +1,17 @@
 purldb-toolkit
 ==============
 
+.. contents:: :local:
+    :depth: 3
+
+
+
 purldb-toolkit is command line utility and library to use the PurlDB, its API and various related libraries.
 
 The ``purlcli`` command acts as a client to the PurlDB REST API end point(s) to expose PURL services.
 It serves both as a tool, as a library and as an example on how to use the services programmatically.
 
- 
+
```

Comment:
`pip install purldb-toolkit` should be in a console section

Reply:
Done -- wrapped `pip install purldb-toolkit` in a `.. code-block:: console` directive.